sor/src/main/java/com/bazaarvoice/emodb/sor/db/astyanax/AstyanaxDeltaIterator.java

```diff
@@ -1,25 +1,21 @@
 package com.bazaarvoice.emodb.sor.db.astyanax;
 
 import com.bazaarvoice.emodb.sor.db.DeltaIterator;
+import com.bazaarvoice.emodb.sor.db.test.DeltaClusteringKey;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'com.bazaarvoice.emodb.sor.db.test.DeltaClusteringKey'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.qa.bazaarvoice.com/coding_rules#rule_key=squid%3AUselessImportCheck)